CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [0.6.0](https://github.com/inviqa/harness-go/tree/0.6.0) (2021-02-01)
+
+[Full Changelog](https://github.com/inviqa/harness-go/compare/0.5.1...0.6.0)
+
+**Implemented enhancements:**
+
+- Update external-images CI pull to skip pulling images locally existing [\#67](https://github.com/inviqa/harness-go/pull/67) ([andytson-inviqa](https://github.com/andytson-inviqa))
```

Comment:
just out of curiosity why not  `@andtson-inviqa`? it gets linked automatically when viewed on GH

Reply:
good question, this is generated using the [Github Changelog Generator](https://github.com/github-changelog-generator/github-changelog-generator)

Reply:
I guess it is more portable this way (e.g. clickable links when viewed outside of GitHub)